### PR TITLE
#7224 Pull images parallel during deployment

### PR DESCRIPTION
**Pull request by AshwinMN24**
We had first cleaned the docker images from the development env servers before deploying. by using below commands.
docker stack ls
NAME       SERVICES
opencrvs   34
 docker stack rm opencrvs
docker system prine -af

then had triggered the github action with the code changes from https://github.com/opencrvs/opencrvs-farajaland/actions/runs/10938830700/job/30369325656 and then repeated the cleanup process and triggered with the old code changes. https://github.com/opencrvs/opencrvs-farajaland/actions/runs/10937970094 

As per the pipeline triggered it had taken less time.

### CODE REVIEW

**Comment by n1koo** [comment]
Looks good but how was this tested? Can you also describe the intent in few lines on the `description`?

Longer term we should also have error control here - eg if pull failed we shouldnt move forward. Right now it will just hammer through, but this change doesn't really make it any worse just maintains that

File: infrastructure/deployment/deploy.sh [hunk 1]
```diff
@@ -289,9 +289,11 @@ docker_stack_deploy() {
     do
       echo "Server failed to download $tag. Retrying..."
       sleep 5
-    done
+    done &
   done
-
+  echo "Images are getting downloaded"
```
Comment (n1koo) [inline infrastructure/deployment/deploy.sh:294]: Would this disappear in the log output of the pull(s)? Eg are they outputting earlier and parallel so this gets lost in their log output...?

If so we can drop this

Comment (AshwinMN24) [reply infrastructure/deployment/deploy.sh:294]: No I could see it as i have rephrased the sentence "Images download is in process" to "Images are getting downloaded" if you check https://github.com/opencrvs/opencrvs-farajaland/actions/runs/10938830700/job/30369325656 you could see it after downloaded the images

Comment (rikukissa) [inline infrastructure/deployment/deploy.sh:294]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@
 ### Improvements
 
 - Auth token, ip address, remote address, mobile number, email redacted/masked from server log
+- Improved deployment time by images download faster.
```
Comment (rikukissa) [inline CHANGELOG.md:18]:
```suggestion
- Optimised deployment times by making docker images download in parallel.
```

**Pull request merged**